R/xml_find.R

```diff
@@ -203,3 +207,13 @@ xml_find_lgl.xml_nodeset <- function(x, xpath, ns = xml_ns(x)) {
 xml_find_lgl.xml_missing <- function(x, xpath, ns = xml_ns(x)) {
    logical(0)
 }
+
+# Deprecated functions ----------------------------------------------------
+
+#' @rdname xml_find_all
+#' @usage NULL
+#' @export
+xml_find_one <- function(x, xpath, ns = xml_ns(x)) {
+  .Deprecated("xml_find_first")
```

Comment:
OTOH a message won't force anyone to notice the change. I can probably find time to fix svglite.  Maybe we give people a month?  That would make a CRAN release around May 27, which would line up nicely with useR.